src/components/trade/LimitOrder/LimitOrder.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { LimitOrder, Props } from './LimitOrder'
+import BigNumber from 'bignumber.js'
+
+export default {
+  title: 'Trade/LimitOrder',
```

Comment:
:book: :cry:

Reply:
I will add it in any missing story in this PR